### PR TITLE
Test PR 18599

### PR DESCRIPTION
**Pull request by luzpaz**
* https://github.com/FreeCAD/FreeCAD/pull/18599

### CODE REVIEW

**Comment by luzpaz** [comment]
Looks like this functionality is broken ATM, @adrianinsaval

**Comment by luzpaz** [comment]
Note:

> ubuntu-latest pipelines will use ubuntu-24.04 soon. For more details, see https://github.com/actions/runner-images/issues/10636

**Comment by adrianinsaval** [comment]
that's probably not important right now, the actual error in the log says it couldn't run apt, might just be some transient problem, can you re run the action?
```
Installing package repositories...
Refreshing package repositories...
Failed to refresh package list: failed to run apt update.
```

**Comment by luzpaz** [comment]
Retriggered action, same result.

**Comment by luzpaz** [comment]
Tested retriggering via editing the PR in https://github.com/FreeCAD/FreeCAD-snap/pull/140/commits/2bdcc0e33005583c6be1802a8cda59c6aa5392be  
No joy :(

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -153,7 +153,8 @@ parts:
 
   freecad:
     plugin: cmake
-    source: https://github.com/FreeCAD/FreeCAD.git
+    source: https://github.com/luzpaz/FreeCAD.git
+    source-branch: PR18599
```
Comment (adrianinsaval) [inline snap/snapcraft.yaml:158]: This doesn't look right, I can't find that branch in your fork

Comment (adrianinsaval) [reply snap/snapcraft.yaml:158]: I believe this should be

```suggestion
    source: https://github.com/ein-shved/FreeCAD.git
    source-branch: about-thumbnail
```

**Comment by furgo16** [comment]
Are snaps build from these type of PRs automatically uploaded to the store? If so, on which channel?

File: snap/snapcraft.yaml (hunk 1 above)
Comment (luzpaz) [reply snap/snapcraft.yaml]: :facepalm:

**Comment by luzpaz** [comment]
The PR has been superceded by https://github.com/FreeCAD/FreeCAD/pull/18617  
I'm going to test this with a new PR.

@furgo16 when they are completed they will have their own temporary channel. you can look at previous PRs that have been closed. or just wait till I build a proper PR that works.

**Comment by luzpaz** [comment]
@furgo16 see https://github.com/FreeCAD/FreeCAD-snap/pull/44#issuecomment-1203371090

**Comment by furgo16** [comment]
Nice, thanks

**Pull request closed without merging**